packages/bluesnap-direct-integration/src/validation-schemas/getEcpValidationSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { memoize } from '@bigcommerce/memoize';
+import { object, ObjectSchema, string } from 'yup';
+
+export default memoize(function getEcpValidationSchema(): ObjectSchema {
+    return object({
+        accountNumber: string()
+            .required('Account Number is required')
+            .matches(/^\d+$/, 'Account Number must contain only numbers')
+            .min(8, 'Account Number must have at least 8 digits'),
+        routingNumber: string()
+            .required('Routing Number is required')
+            .matches(/^\d+$/, 'Routing Number must contain only numbers')
+            .length(9, 'Routing Number must be 9 digits long'),
+        accountType: string().required('Account Type is required'),
```

Comment:
Do we need to look into translating these?

Reply:
Please see comments above. https://github.com/bigcommerce/checkout-js/pull/1218#discussion_r1147850530 and https://github.com/bigcommerce/checkout-js/pull/1218#discussion_r1123709898